policyengine_us/parameters/gov/states/mn/tax/income/niit/rate.yaml

```diff
@@ -0,0 +1,22 @@
+description: Minnesota taxes net investment income at the following rates.
+brackets:
+  - threshold:
+      2024-01-01: 0
+    rate:
+      2024-01-01: 0
+  - threshold:
+      2024-01-01: 1_000_000
+    rate:
+      2024-01-01: 0.01
```

Comment:
The tests fail because it is expecting values for previous years (pre-2024) - we will need to add an `in_effect.yaml` and apply it to `mn_income_tax_before_credits` 

2021:false - does not add `mn_niit`
2024: true - applies `mn_niit`